common/styles/base/_wellcome_normalize.scss

```diff
@@ -8,3 +8,11 @@ fieldset {
   margin: 0;
   padding: 0;
 }
+
+// removes up and down arrows webkit adds to number inputs on desktop
+input[type=number]::-webkit-inner-spin-button,
```

Comment:
is there any accessibility issues with this?

Reply:
No, there are still a plethora of ways for users to input dates